src/Utility/Function.h

```diff
@@ -262,6 +269,19 @@ class VecFunc : public utl::SpatialFunction<Vec3>, public FunctionBase
     result = this->evalGradient(X);
     return result;
   }
+
+  //! \brief Evaluates time derivatives of the function.
+  Vec3 timeDerivative(const Vec3& X) const
+  {
+    return this->evalTimeDerivative(X);
+  }
```

Comment:
Define on single line

Reply:
single-lined although it gets to 82 chars.

Reply:
yep, that's the price of using a bit long'ish method names